soroban-sdk-macros/src/derive_client.rs

```diff
@@ -118,10 +124,16 @@ pub fn derive_client(crate_path: &Path, ty: &str, name: &str, fns: &[syn_ext::Fn
         pub struct #client_ident<'a> {
             pub env: #crate_path::Env,
             pub contract_id: #crate_path::BytesN<32>,
+            #[doc(hidden)]
```

Comment:
Slipping this doc hidden in so that in so that in IDEs these generated non-public fields don't display in auto-complete for developers. Even though they aren't public, because this code gets generated inside a developer's module, the developer can access the private fields and them showing up in auto-completes is very annoying.